File: src/main/java/org/example/lab9/Figure.java
package org.example.lab9;

import org.example.lab8.Body;

public class Figure<T extends Body> implements Interface<T>  {
    private T figure;
    public Figure(T figure){
        this.figure=figure;
    }

    @Override
    public double findArea() {
        return figure.FindArea();
    }

    @Override
    public double findVolume() {
        return figure.FindVolume();
    }
}

File: lab1C/lab9/Figure.cs
using lab1C.lab8;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab1C.lab9
{
   class Figure<T> : Interface<T> where T : Body
    {
        private T figure;

        public Figure(T figure)
        {
            this.figure = figure;
        }

        public double findArea()
        {
            return figure.FindArea();
        }

        public double findVolume()
        {
            return figure.FindVolume();
        }
    }
}
